Ollivanders/src/net/pottercraft/ollivanders2/Ollivanders2.java

```diff
@@ -753,6 +779,9 @@ private void usageMessageOllivanders(@NotNull CommandSender sender)
      */
     private boolean runHouse(@NotNull CommandSender sender, @NotNull String[] args)
     {
+        if (!sender.hasPermission("Ollivanders2.admin"))
```

Comment:
These should never happen because runOllivanders is also checking. I added them as an extra safety check just in case.